sdks/java/io/common/src/test/java/org/apache/beam/sdk/io/common/IOTestPipelineOptions.java

```diff
@@ -45,19 +45,16 @@
   void setBigQueryTable(@Nullable String tableName);
 
   @Description("InfluxDB measurement to publish results to.")
-  @Nullable
   String getInfluxMeasurement();
 
   void setInfluxMeasurement(@Nullable String measurement);
 
   @Description("InfluxDB host.")
-  @Nullable
   String getInfluxHost();
 
   void setInfluxHost(@Nullable String host);
 
   @Description("InfluxDB database.")
-  @Nullable
```

Comment:
This change is curious. The `setInfluxDatabase` method accepts `null`, so this value should also be nullable. Should you change the setter as well? Or is there something else going on?

Reply:
I removed those nullable as it was expecting these fields non-null [here](https://github.com/jayendra13/beam/blob/10620/sdks/java/testing/test-utils/src/main/java/org/apache/beam/sdk/testutils/publishing/InfluxDBSettings.java#L31) via, [this](https://github.com/jayendra13/beam/blob/10620/sdks/java/extensions/sql/perf-tests/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/bigquery/BigQueryIOPushDownIT.java#L114). I removed the `Nullable` from here thinking of it as a test code and [InfluxDBSettings.java](https://github.com/jayendra13/beam/blob/10620/sdks/java/testing/test-utils/src/main/java/org/apache/beam/sdk/testutils/publishing/InfluxDBSettings.java) as a non-test code.
Annotating the corresponding fields at [InfluxDBSettings.java](https://github.com/jayendra13/beam/blob/10620/sdks/java/testing/test-utils/src/main/java/org/apache/beam/sdk/testutils/publishing/InfluxDBSettings.java) as `Nullable` also should work.
Should I go with that?

Reply:
Ah that makes sense. What you should do, I think, is alter `setInfluxDatabase` so the parameter is non-nullable. Otherwise somehow the pipeline options would be magically turning `null` into non-`null`.

Reply:
So these three getters (for measurements, host and database) are fine, and we should also change these three setters to take `non-null` arguments. Correct me if I am wrong.

Reply:
Yes, exactly. I wonder if we can alter the core `PipelineOptions` code that generates the proxy objects to ensure that the nullability matches... (if you want an extra fun follow up project...)